app/assets/javascripts/discourse/app/templates/components/categories-boxes-with-topics.hbs

```diff
@@ -4,8 +4,8 @@
       <div class="category-box-heading">
         <a href={{c.url}}>
           {{#unless c.isMuted}}
-            {{#if c.uploaded_logo.url}}
-              <CdnImg @src={{c.uploaded_logo.url}} @class="logo" @width={{c.uploaded_logo.width}} @height={{c.uploaded_logo.height}} @alt="" />
+            {{#if (or c.uploaded_logo.url c.uploaded_logo_dark.url)}}
+              <CategoryLogo @category={{c}} @class="logo" @alt="" />
```

Comment:
`@alt=""` can be removed here, as far as I can tell, it doesn't do anything. (This wasn't great previously either, but since the `CategoryLogo` component is not an image at all, you don't need to pass that empty string.